project-koku/koku
# [COST-4423] - Initial work for cluster/source info page

## Jira Ticket

[COST-4423](https://issues.redhat.com/browse/COST-4423)

## Description

This change will add all the additional info required for building the cluster info page.

## Testing

1. Checkout Branch
2. Restart Koku
3. make create/load customer data
4. Check the sources endpoint `/api/cost-management/v1/sources/` and specific sources `/api/cost-management/v1/sources/1/`
5. See that we now have the additional details filled in depending on the source type, the below is an OCP example.

```
last_payload_received_at": "2024-02-19T14:22:10.521247Z",
            "status": {
                "download": "complete",
                "processing": "complete",
                "summary": "complete"
            },
            "infrastructure": {
                "type": "GCP-local",
                "uuid": "a1cfd795-3bd0-4a38-a79e-5644b827d07d",
                "id": 4,
                "account": "todays-raw-calc-id",
                "region": "us-west1-a"
            },
            "additional_context": {
                "operator_version": "4.4.9",
                "operator_airgapped": false,
                "operator_certified": false,
                "operator_update_available": true
            }
```

## Notes
MIGRATIONS - https://github.com/project-koku/koku/pull/4932
...
